requirements.txt

```diff
@@ -1,7 +1,8 @@
 django>=3.2,<4.0
 django-cas-ng
 pucas>=0.6
-parasolr>=0.8.2
+#parasolr>=0.8.2
+git+https://github.com/Princeton-CDH/parasolr@feature/grouped-response#egg=parasolr  # @NOTE: For testing, allowing page grouping
```

Comment:
Should we get rid of this? We're gonna merge that PR soon?

Reply:
good point. if we're satisfied that the parasolr part is working and stable, we should merge that and update this to install from `develop` (and then we'll release a new version of parasolr before the new ppa release, after the testing — just in case acceptance testing identifies something we missed)